authapi/serializers.py

```diff
@@ -51,11 +51,46 @@ class UserSerializer(serializers.ModelSerializer):
         many=True, source='seedteam_set', read_only=True)
     organizations = OrganizationSummarySerializer(
         many=True, source='seedorganization_set', read_only=True)
-    email = serializers.EmailField(source='username')
-    admin = serializers.BooleanField(source='is_superuser')
+    email = serializers.EmailField()
+    admin = serializers.BooleanField(source='is_superuser', required=False)
```

Comment:
Ah, true. Hmm, a `PUT` request where not all non-defaulting fields are required feels like a partial update. Any reason we don't just have this field be required?